modules/apps/journal/journal-service/src/main/java/com/liferay/journal/service/impl/JournalArticleLocalServiceImpl.java

```diff
@@ -1989,6 +1989,53 @@ public JournalArticle fetchLatestArticleByExternalReferenceCode(
 			groupId, externalReferenceCode, orderByComparator);
 	}
 
+	/**
+	 * Returns the latest web content article matching the group, the
+	 * external reference code and the status.
+	 *
+	 * @param  groupId the primary key of the web content article's group
+	 * @param  externalReferenceCode the web content article's external
+	 *         reference code
+	 * @param  status the web content article's workflow status. For more
+	 *         information see {@link WorkflowConstants} for constants starting
+	 *         with the "STATUS_" prefix.
+	 * @param  preferApproved whether to prefer returning the latest matching
+	 *         article that has workflow status {@link
+	 *         WorkflowConstants#STATUS_APPROVED} over returning one that has a
+	 *         different status
+	 * @return the latest matching web content article, or <code>null</code> if
+	 *         no matching web content article could be found
+	 */
+	@Override
+	public JournalArticle fetchLatestArticleByExternalReferenceCode(
+		long groupId, String externalReferenceCode, int status,
+		boolean preferApproved) {
```

Comment:
The current pattern is for the ERC to come first in the parameter list: fetch*ByExternalReferenceCode(externalReferenceCode, groupId, status, ...).

I see there are other methods with the old ordering in this class, so it may be ok to use the older way for consistency sake. But it is possible that Brian will ask you to change it, so I'd comment about it in the PR description, something like:
"For consistency, I'm using the old ordering for the ERC in the methods. Let me know if we should change it consistently; we'll send that in a followup pr"

Reply:
OK, I see , Thanks for the recommendation